asciigraf/asciigraf.py

```diff
@@ -15,10 +15,10 @@ def graph_from_ascii(network_string):
     """ Produces a networkx graph, based on an ascii drawing
         of a network
     """
-    EDGE_CHAR_NEIGHBOURS = {
+    EDGE_CHAR_NEIGHBOURS = {  # first point in tuple is the point parsed first
         "-":  [Point(-1, 0), Point(1, 0)],
         "\\": [Point(-1, -1),  Point(1, 1)],
-        "/":  [Point(-1, 1), Point(1, -1)],
```

Comment:
I wanted to order all the points so the first one would always be the one we'd already have parsed. So that it's the "leading" neighbour. It's sort of a fussy detail.